bedrock/mozorg/templates/mozorg/contact/contact-landing.html

```diff
@@ -28,7 +28,7 @@
       <p>
       {% trans link_sumo='https://support.mozilla.org' %}
         To reach Mozilla anywhere in the world, check out the locations above. For help
-        with any of our products, including Firefox for desktop and Android, Firefox OS
+        with any of our products, including Firefox (for desktop, Android and iOS), Firefox OS
```

Comment:
They opted to keep FxOS here because folks using those devices might still come looking for help.